tests/integration/test_definitions.py

```diff
@@ -316,13 +316,13 @@ def define_tests():
         "black check merlin": (f"{black} merlin/", HasReturnCode(), "local"),
         "black check tests": (f"{black} tests/", HasReturnCode(), "local"),
     }
-    dependency_checks = {
-        "deplic no GNU": (
-            "deplic ./",
-            [HasRegex("GNU", negate=True), HasRegex("GPL", negate=True)],
-            "local",
-        ),
-    }
+    # dependency_checks = {
```

Comment:
why commented out?

Reply:
Just a relic of experimenting with the pipeline, it's back now - the Actions log _seemed_ to indicate that some of the pipelines were hanging on the dependency check before they even got to the distributed test (it wasn't, it was hanging on the distributed test).